src/test/java/com/onfido/integration/WorkflowRunOutputsTest.java

```diff
@@ -0,0 +1,114 @@
+    WorkflowRun workflowRun = onfido.findWorkflowRun(workflowRunId);
+
+    Map<String, Object> output = (Map<String, Object>) workflowRun.getOutput();
+    Assertions.assertEquals(completeTaskRequest.getData(), output.get("profile_capture_data"));
+  }
+
+  @Test
+  public void documentAndFacialSimilarityReportsAsOutputs() throws Exception {
+    UUID workflowId = UUID.fromString("5025d9fd-7842-4805-bce1-a7bfd7131b4e");
+    UUID workflowRunId = createWorkflowRun(workflowId, applicant.getId()).getId();
+
+    String profileDataTaskId = getTaskIdByPartialId(workflowRunId, "profile");
+
+    Map<String, String> completeProfileDataTaskBody = new HashMap<>();
+    completeProfileDataTaskBody.put("first_name", "First");
+    completeProfileDataTaskBody.put("last_name", "Last");
+
+    CompleteTaskRequest completeProfileDataTaskRequest = new CompleteTaskRequest();
+    completeProfileDataTaskRequest.setData(completeProfileDataTaskBody);
+    onfido.completeTask(workflowRunId, profileDataTaskId, completeProfileDataTaskRequest);
+
+    String documentCaptureTaskId = getTaskIdByPartialId(workflowRunId, "document_photo");
+    Document document = uploadDocument(applicant, "sample_driving_licence.png", "driving_licence");
+
+    Map<String, String> completeDocumentCaptureTaskBody = new HashMap<>();
+    List<Object> completeDocumentCaptureTaskBodyArray = new ArrayList<>();
+    CompleteTaskRequest completeDocumentCaptureTaskRequest = new CompleteTaskRequest();
+
+    completeDocumentCaptureTaskBody.put("id", document.getId().toString());
+    completeDocumentCaptureTaskBodyArray.add(completeDocumentCaptureTaskBody);
+    completeDocumentCaptureTaskRequest.setData(completeDocumentCaptureTaskBodyArray);
+    onfido.completeTask(workflowRunId, documentCaptureTaskId, completeDocumentCaptureTaskRequest);
+
+    String photoCaptureTaskId = getTaskIdByPartialId(workflowRunId, "face_photo");
+    LivePhoto livePhoto = uploadLivePhoto(applicant, "sample_photo.png");
+
+    Map<String, String> completeLivePhotoCaptureTaskBody = new HashMap<>();
+    List<Object> completeLivePhotoCaptureTaskBodyArray = new ArrayList<>();
+    CompleteTaskRequest completeLivePhotoCaptureTaskRequest = new CompleteTaskRequest();
+
+    completeLivePhotoCaptureTaskBody.put("id", livePhoto.getId().toString());
+    completeLivePhotoCaptureTaskBodyArray.add(completeLivePhotoCaptureTaskBody);
+    completeLivePhotoCaptureTaskRequest.setData(completeLivePhotoCaptureTaskBodyArray);
+    onfido.completeTask(workflowRunId, photoCaptureTaskId, completeLivePhotoCaptureTaskRequest);
+
+    WorkflowRun workflowRun = onfido.findWorkflowRun(workflowRunId);
+
+    // wait for workflow run to finish
+    int iteration = 0;
+    while (workflowRun.getStatus().equals(PROCESSING)) {
+      if (iteration > 10) {
+        Assertions.fail("Workflow run did not complete in time");
+      }
+      iteration += 1;
+      Thread.sleep(1000);
+      workflowRun = onfido.findWorkflowRun(workflowRunId);
+    }
+
```

Comment:
Would make sense asserting just out of the loop that workflow run status is `COMPLETE`?